src/features/user/Account/EditModal.tsx

```diff
@@ -91,9 +91,15 @@ export const EditModal = ({
     setDisabled(false);
   }, [editModalOpen]);
 
+  interface bodyToSendType {
```

Comment:
In our codebase we follow PascalCase as a naming convention for all the interfaces